<ui version="4.0" >
 <class>FormMovimientoBase</class>
 <widget class="QWidget" name="FormMovimientoBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>543</width>
    <height>685</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <widget class="QLabel" name="LTitulo" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-size:12pt; font-weight:600;">Agregar Caravanas&lt;/span>&lt;/p>&lt;/body>&lt;/html></string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="label_2" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>Fecha:</string>
       </property>
       <property name="buddy" >
        <cstring>dEFecha</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="dEFecha" >
       <property name="frame" >
        <bool>false</bool>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="displayFormat" >
        <string>dd/MM/yyyy</string>
       </property>
       <property name="calendarPopup" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="LCategoria" >
       <property name="text" >
        <string>Categoria:</string>
       </property>
       <property name="buddy" >
        <cstring>CBCategoria</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="CBCategoria" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="MinimumExpanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="PBAgregarCategoria" >
       <property name="text" >
        <string>Agregar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="LCliente" >
       <property name="text" >
        <string>Vendedor.</string>
       </property>
       <property name="buddy" >
        <cstring>CBCliente</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="CBCliente" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="MinimumExpanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="PBAgregarCliente" >
       <property name="text" >
        <string>Agregar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="LOrigen" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Preferred" hsizetype="Minimum" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>Origen:</string>
       </property>
       <property name="buddy" >
        <cstring>CBEstablecimientoOrigen</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="CBEstablecimientoOrigen" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="MinimumExpanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="PBAgregarEstablecimientoOrigen" >
       <property name="text" >
        <string>Agregar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="LDestino" >
       <property name="text" >
        <string>Destino:</string>
       </property>
       <property name="buddy" >
        <cstring>CBEstablecimientoDestino</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="CBEstablecimientoDestino" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="MinimumExpanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="PBAgregarEstablecimientoDestino" >
       <property name="text" >
        <string>Agregar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" >
     <item>
      <widget class="QLabel" name="LGuia" >
       <property name="text" >
        <string>Numero de Guia:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="LEGuia" />
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <layout class="QHBoxLayout" >
       <item>
        <widget class="QLabel" name="LTri" >
         <property name="text" >
          <string>Numero de TRI:</string>
         </property>
         <property name="buddy" >
          <cstring>LETRI</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="LETRI" >
         <property name="enabled" >
          <bool>false</bool>
         </property>
         <property name="autoFillBackground" >
          <bool>true</bool>
         </property>
         <property name="inputMask" >
          <string/>
         </property>
         <property name="text" >
          <string/>
         </property>
         <property name="alignment" >
          <set>Qt::AlignCenter</set>
         </property>
         <property name="readOnly" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" >
       <item>
        <widget class="QLabel" name="LDTA" >
         <property name="text" >
          <string>Numero DTA:</string>
         </property>
         <property name="buddy" >
          <cstring>LEDTA</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="LEDTA" />
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="GBCaravanas" >
     <property name="title" >
      <string>Caravanas</string>
     </property>
     <layout class="QVBoxLayout" >
      <item>
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QLabel" name="label_8" >
          <property name="text" >
           <string>Numero de caravana:</string>
          </property>
          <property name="buddy" >
           <cstring>LENumCar</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="LENumCar" >
          <property name="dragEnabled" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="PBAgregar" >
          <property name="text" >
           <string>Agregar</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QTableView" name="TVCaravanas" >
          <property name="tabKeyNavigation" >
           <bool>false</bool>
          </property>
          <property name="showDropIndicator" stdset="0" >
           <bool>false</bool>
          </property>
          <property name="dragDropOverwriteMode" >
           <bool>false</bool>
          </property>
          <property name="alternatingRowColors" >
           <bool>true</bool>
          </property>
          <property name="wordWrap" >
           <bool>false</bool>
          </property>
          <property name="cornerButtonEnabled" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" >
          <item>
           <widget class="QPushButton" name="PBEliminar" >
            <property name="text" >
             <string>Eliminar</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="PBDueno" >
            <property name="text" >
             <string>Cambiar 
Due√±o</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="PBCargarArchivo" >
        <property name="text" >
         <string>Cargar desde archivo...</string>
        </property>
       </widget>
      </item>
     </layout>
     <zorder>PBCargarArchivo</zorder>
     <zorder></zorder>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" >
     <item>
      <widget class="QLabel" name="LCantidadAnimales" >
       <property name="text" >
        <string>Cantidad de Animales para esto movimiento</string>
       </property>
       <property name="textInteractionFlags" >
        <set>Qt::NoTextInteraction</set>
       </property>
       <property name="buddy" >
        <cstring>SBCantidadAnimales</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="SBCantidadAnimales" >
       <property name="autoFillBackground" >
        <bool>true</bool>
       </property>
       <property name="wrapping" >
        <bool>false</bool>
       </property>
       <property name="suffix" >
        <string> animales</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>dEFecha</tabstop>
  <tabstop>CBCategoria</tabstop>
  <tabstop>PBAgregarCategoria</tabstop>
  <tabstop>CBCliente</tabstop>
  <tabstop>PBAgregarCliente</tabstop>
  <tabstop>CBEstablecimientoOrigen</tabstop>
  <tabstop>PBAgregarEstablecimientoOrigen</tabstop>
  <tabstop>CBEstablecimientoDestino</tabstop>
  <tabstop>PBAgregarEstablecimientoDestino</tabstop>
  <tabstop>LETRI</tabstop>
  <tabstop>LEDTA</tabstop>
  <tabstop>LENumCar</tabstop>
  <tabstop>PBAgregar</tabstop>
  <tabstop>PBEliminar</tabstop>
  <tabstop>TVCaravanas</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
